src/auth/SSOAuth.ts

```diff
@@ -119,10 +119,27 @@ export default class SSOAuth {
                 this.writer.showError(
                     `Port ${PORT} already in use. Unable to log in.`,
                 )
+                SUPPORTED_PORTS = SUPPORTED_PORTS.filter((p) => p !== PORT)
+                if (!SUPPORTED_PORTS.length) {
+                    this.writer.showError(
+                        'No available ports to start local server',
+                    )
+                    exit(1)
+                }
+                PORT = SUPPORTED_PORTS[0]
+                setTimeout(() => {
+                    this.server.close()
+                    this.server.listen(PORT, host, () => {
```

Comment:
are you sure there isn't a way to just tell it to listen on "any available open port"? Usually if you just pass 0 or something then it will do that

Reply:
our auth is setup to only accept the supported ports as redirects

Reply:
so using any port will unfortunately error

Reply:
gotcha